_posts/2014-10-02-sinatra-app-tutorial.markdown

```diff
@@ -39,9 +39,9 @@ You can actually call your Ruby file whatever you'd like. `vote.rb` for instance
 ### Run your app
 
 Go to the directory where you put your app and run `ruby suffragist.rb`.
-Now you can visit [localhost:4567](http://localhost:4567). You should
+Now you can visit <a href="localhost:4567" target="_blank">localhost:4567</a>. You should
 see a ‘Hello, voter!’ page, which means that the generation of your new
-app worked correctly. Hit `ctrl-c` in the terminal to quit the server.
+app worked correctly. Hit `CTRL + C` in the terminal to shut down the server. If `CTRL + C` does not work for you it means you are probably Windows user and `CTRL + Z`/ `Ctrl + Pause` / `Ctrl + Pause` will fix the issue)
```

Comment:
I'll not comment on whether the Microsoft version should be used by default, but I think you can still leave out the spaces.